status-im/status-react
# [BUG #1990] Improved amount validation in wallet/send screen

fixes #1990

### Summary:
Fixes incorrect amount validation in wallet/send screen

### Steps to test:
- Open Status
- Navigate to wallet/send screen
- Enter an amount bigger than wallet balance: direct signing should be disabled while later should be enabled

status: ready
